code/game/turfs/closed/wall/window_frames.dm

```diff
@@ -8,6 +8,7 @@
 	smoothing_flags = SMOOTH_BITMASK|SMOOTH_OBJ
 	smoothing_groups = list(SMOOTH_GROUP_WINDOW_FRAMES)
 	canSmoothWith = list(SMOOTH_GROUP_WINDOW_FRAMES)
+	pass_flags_self = PASSTABLE | LETPASSTHROW
```

Comment:
what'd they have before?

Reply:
default which is null

Reply:
This might behave weirdly with dwarfs/monkys, you mind checking?

Reply:
Checked on local, dwarves and monkeys are able to walk over the low walls as they stand. Just the low walls work fine, 
![image](https://user-images.githubusercontent.com/41715314/219832561-6733616c-64ec-47ce-9274-7c1c16329097.png)
Placing window glass on top by itself appears to cause visual artifacting to get left over on the mobile glass.
![image](https://user-images.githubusercontent.com/41715314/219832948-1cbff015-0af6-4065-a83b-e0bafe59447b.png)
However, going through the construction steps appears to still behave fine. wall is solid and intact. No inconsistent throw behavior either.

In retrospect, the window artifacting is apparently just specific to windows... so the low walls looks good to go.